ledgers/xero/bin/xero_oauth_server.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+# Setup
+#
+# gem install bundler
+# Ensure you have http://localhost:5678 (or PORT) as a Redirect URI in QBO.
+
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'dotenv'
+  gem 'ledger_sync'
+  gem 'ledger_sync-xero', path: '../'
+  gem 'rack'
+  gem 'pd_ruby'
+end
+
+puts 'Gems installed and loaded!'
+
+require 'pd_ruby'
+require 'socket'
+require 'dotenv'
+require 'rack'
+require 'ledger_sync'
+require 'rack/lobster'
```

Comment:
We do not need this line anymore.  It actually shouldn't be in the old one.  It was from the original scrip I modeled this off of.

Also, is there a way to DRY these up between the two scripts?

Reply:
Yea, I'll clean it up.